var/spack/repos/builtin/packages/py-espresso/package.py

```diff
@@ -39,7 +39,9 @@ class PyEspresso(CMakePackage):
     """
     homepage = "http://espressomd.org/"
     git      = "https://github.com/espressomd/espresso.git"
+    url      = "https://github.com/espressomd/espresso/releases/download/4.0.0/espresso-4.0.0.tar.gz"
 
+    version('4.0.0', sha256='8e128847447eebd843de24be9b4ad14aa19c028ae48879a5a4535a9683836e6b')
```

Comment:
conventionally `develop` is the first version

Reply:
Done